l10n_it_fatturapa_out/wizard/efattura.py

```diff
@@ -261,7 +261,24 @@ def to_xml(self, env):
             # i controlli precedenti dovrebbero escludere errori di sintassi XML
             # with open("/tmp/fatturaout.xml", "wb") as o:
             #    o.write(etree.tostring(root, xml_declaration=True, encoding="utf-8"))
-            raise UserError("\n".join(str(e) for e in errors))
+            # Print error paths, as they can be helpful even for non-technical people
+            error_path_string = "\n- ".join(
+                err.path[err.path.index(":") + 1 :] for err in errors
+            )
+            error = _(
+                "Error processing invoice(s) %(invoices)s.\n\n"
+                "Errors in the following fields:\n- %(error_path_string)s",
+                invoices=", ".join(
+                    inv.display_name for inv in template_values["invoices"]
+                ),
+                error_path_string=error_path_string,
+            )
+            # add details in debug mode
+            if self.env.user.has_group("base.group_no_one"):
```

Comment:
In un `else`, aggiungerei qualcosa tipo "Activate debug mode to see the full error", che ne pensi?